tests/neg/i6779.check

```diff
@@ -8,8 +8,9 @@
    |                           ^
    |                           Found:    F[T]
    |                           Required: F[G[T]]
--- [E007] Type Mismatch Error: tests/neg/i6779.scala:13:31 -------------------------------------------------------------
+-- [E006] Not Found Error: tests/neg/i6779.scala:13:27 -----------------------------------------------------------------
 13 |def g3[T](x: T): F[G[T]] = f(x)(using summon[Stuff]) // error
```

Comment:
The test `tests/neg/i6779.scala` should be changed to have
```scala
def g3[T](x: T): F[G[T]] = extension_f(x)(using summon[Stuff]) // error
```

That produces the same error message as before (maybe with different position)